crowdsec-docs/docs/appsec/installation.md

```diff
@@ -0,0 +1,130 @@
+
+```bash
+mkdir  -p /etc/crowdsec/acquis.d
+cat > /etc/crowdsec/acquis.d/appsec.yaml << EOF
+listen_addr: 127.0.0.1:4242
+appsec_config: crowdsecurity/vpatch
+source: appsec
+labels:
+  type: appsec
+EOF
+```
+
+The important lines are `listen_addr` that indicates on which interface/port the service listens too, and `appsec_config` that tells which configuration the appsec component will run.
+
+Restart crowdsec:
+
+```bash
+systemctl restart crowdsec
+```
+
+You should be able to see crowdsec starting the aplication security component in the logs (`/var/log/crowdsec.log`):
+
+```
+INFO[2023-12-05 09:16:31] 1 appsec runner to start                      type=appsec
+INFO[2023-12-05 09:16:31] Starting Appsec server on 127.0.0.1:4242/     type=appsec
+INFO[2023-12-05 09:16:31] Appsec Runner ready to process event          type=appsec uuid=3b80fefe-6665-4f81-8567-a2a7f09a706a
+```
+
+and actively listening on the port:
+
+```bash
+# netstat -laputen | grep 4242    
+tcp        0      0 127.0.0.1:4242          0.0.0.0:*               LISTEN      0          6923691    779516/crowdsec     
+
+```
+
+# Configuration : Remediation component
+
+<!-- @kka fix version -->
+At the time of writting, only the crowdsec nginx bouncer supports native integration with the AppSec component, and requires version >= X.Y.Z.
+
+[If you don't have the bouncer installed, do it now !](https://docs.crowdsec.net/u/bouncers/nginx)
+
+To enable the integration, you simply have to add a `APPSEC_URL` parameter to the existing nginx bouncer remediation configuration :
+
+> /etc/crowdsec/bouncers/crowdsec-nginx-bouncer.conf
+```
+...
+APPSEC_URL=http://127.0.0.1:4242
+...
+```
+
+:warning: The remediation component authenticates to the AppSec component using the same API KEY it uses to authenticate to LAPI. Your bouncer must thus have a valid api key for the LAPI the security engine running the AppSec component is connected to :warning:
+
+We can now restart our remediation component:
+
+```bash
+sudo systemctl restart nginx
+```
+
+# Testing
+
+We can try to trigger a rule that is part of the [CISA Virtual Patching Collection](https://hub.crowdsec.net), for example the rule for `CVE-2023-42793`. It is trivial, as it only requires us to try to access a URI that ends with `/rpc2`: <!-- @tko : fix link to collec when merged -->
+
+```
+▶ curl -I localhost/rpc2 
+HTTP/1.1 403 Forbidden
+Server: nginx/1.18.0 (Ubuntu)
+Date: Tue, 05 Dec 2023 14:26:03 GMT
+Content-Type: text/html
+Connection: keep-alive
+```
+
+And if we look at it in a browser, the user is presented with the HTML page emitted by the remediation component (that can be customized):
```

Comment:
that can be customized. do we have a page about this customization? we could link to it

Reply:
@blotus on a ça ?